functests/utils/images/images.go

```diff
@@ -0,0 +1,54 @@
+package images
+
+import (
+	"fmt"
+	"os"
+
+	gomega "github.com/onsi/gomega"
+)
+
+var registry string
+var images map[string]imageLocation
+
+const (
+	// TestUtils is the image name to be used to retrieve the test utils image
+	TestUtils = "testutils"
```

Comment:
should the name here start with an upper case?

Reply:
It's the key constant, I guess it's irrelevant.
I think it's a matter of taste, but since in the perf operator we left it lowercase and it's merged, I would leave lowercase for consistency.